targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_064S2_4343W/device/COMPONENT_CM4/TOOLCHAIN_ARM/cyb06xxa_cm4.sct

```diff
@@ -27,6 +27,7 @@
 ;*******************************************************************************
 ;* \copyright
 ;* Copyright 2016-2019 Cypress Semiconductor Corporation
+;* Copyright 2020 Arm Limited
```

Comment:
Why isn't a similar change necessary for the CY8CPROTO_064_SB target?

Reply:
What are the long-term plans for `CY8CPROTO_064_SB`? Will Cypress be removing `CY8CPROTO_064_SB` from Mbed OS, along with `CY8CKIT_064S2_4343W`, when final hardware is available?

Reply:
CY8CPROTO_064_SB is not supported by TF-M (master). So we’ll not be able to support that target for PSA in Mbed OS.